openedx/core/djangoapps/credit/api/eligibility.py

```diff
@@ -316,6 +316,9 @@ def remove_credit_requirement_status(username, course_key, req_namespace, req_na
         )
         return
 
+    # There should be at most one requirement
+    req_to_remove = req_to_remove[0]
```

Comment:
Should this function just call `get_course_requirement()` instead of `get_course_requirements()` to fetch `req_to_remove`?  It looks like that would have the same effect, by using a more appropriate ORM method.

Reply:
Hm, I didn't notice that function existed, will take a look.